### ISSUE TITLE
Model is not converging... 

### ISSUE THREAD

**Issue by isaac-223** [facebookresearch/deit#72]
Hello,
I am trying to train the custom dataset consist of 450 images per class, total classes are 6., I have tried to change the learning rate to 0.0003, 0.003 etc. also changed the nb_classes to 6 in dataset.py but the loss doesnt seem to go down and accuracy doesnt seem to go up than 16.67%? what could be the issue?
The model achieved 16.67% in second epoch but its not going forward...

python main.py --model deit_tiny_patch16_224 --batch-size 64 --lr 0.1 --epochs 1000 --data-path /mnt/2tb/general/deit/500_aug_data --output_dir /mnt/2tb/general/deit/output_lr_0.1

Thanks in advance,
Isaac.
